store/pipelineRecord.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2017 caicloud authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package store
+
+import (
+	"time"
+
+	"github.com/caicloud/cyclone/pkg/api"
+	"gopkg.in/mgo.v2/bson"
+)
+
+// CreatePipelineRecord creates the pipelineRecord, returns the pipelineRecord created.
+func (d *DataStore) CreatePipelineRecord(pipelineRecord *api.PipelineRecord) (*api.PipelineRecord, error) {
+	pipelineRecord.ID = bson.NewObjectId().Hex()
+	pipelineRecord.StartTime = time.Now()
+
+	err := d.pipelineRecordCollection.Insert(pipelineRecord)
```

Comment:
No need to declare the variable `err`, change to: 
```
if err := d.pipelineRecordCollection.Insert(pipelineRecord); err != nil {
  return nil, err
}
```
Please check the same error in other places.